### PR TITLE
[PRM-2313] Added TPP content to LPP1

### PR DESCRIPTION
**Pull request by JBlacklaws**
Trader:
Screenshot 2023-02-20 at 11 31 36
Agent:
Screenshot 2023-02-20 at 11 32 14

### CODE REVIEW

**Comment by JBlacklaws** [comment]
Requires: https://github.com/hmrc/penalties/pull/237

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Could you rebase please?

File: app/viewmodels/CalculationPageHelper.scala [hunk 1]
```diff
@@ -58,8 +58,8 @@ class CalculationPageHelper @Inject()(implicit val appConfig: AppConfig) extends
     penaltyDetails.latePaymentPenalty.exists {
       _.details.exists { //Current understanding is that TTP values are replicated across every LPP
         _.LPPDetailsMetadata.timeToPay.exists {
-          _.exists(ttp => {
-            if(ttp.TTPEndDate.isDefined) {
+          value => value.exists(ttp => {
```
Comment (LewisMitchell) [inline app/viewmodels/CalculationPageHelper.scala:61]: I think this can be reverted back to how it was

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: it/controllers/CalculationControllerISpec.scala [hunk 2]
```diff
@@ -393,6 +393,46 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase with FeatureS
       ))
     ))
   )
+
+  val penaltyDetailsWithDay15ChargeTPPActive: GetPenaltyDetails = samplePenaltyDetails.copy(
```
Comment (LewisMitchell) [inline it/controllers/CalculationControllerISpec.scala:397]:
```suggestion
  val penaltyDetailsWithDay15ChargeTTPActive: GetPenaltyDetails = samplePenaltyDetails.copy(
```

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**